card/src/main/java/com/adyen/checkout/card/BinLookupData.kt

```diff
@@ -0,0 +1,16 @@
+package com.adyen.checkout.card
+
+/**
+ * Class that contains data about the entered card number.
+ *
+ * @param brand The detected brand.
+ * @param paymentMethodVariant The type of payment method. See the
+ * [docs](https://docs.adyen.com/development-resources/paymentmethodvariant/) for more information.
+ * @param isReliable Indicates whether the data is reliable. If true, the card number is checked with Adyen systems.
+ * If false, the data is provided by a local detection algorithm.
+ */
+data class BinLookupData(
+    val brand: String,
+    val paymentMethodVariant: String?,
```

Comment:
Maybe we should call this `paymentMethodType`, since it's called `type` on the API

Reply:
There is a [docs page](https://docs.adyen.com/development-resources/paymentmethodvariant/) on this field and web also uses this name. Are you sure we should change it?

Reply:
Nevermind then 👍